--- origsrc/ORBit2-2.14.19/linc2/src/Makefile.am	2010-02-09 06:05:35.000000000 -0600
+++ src/ORBit2-2.14.19/linc2/src/Makefile.am	2013-11-19 04:23:02.665515900 -0600
@@ -12,8 +12,7 @@ INCLUDES =                         \
 	-I$(top_builddir)/linc2/include  \
 	-I$(top_srcdir)/linc2/include    \
 	$(LINC_CFLAGS)             \
-	$(WARN_CFLAGS)             \
-	-DG_DISABLE_DEPRECATED
+	$(WARN_CFLAGS)
 
 if OS_WIN32
 else
